### PR TITLE
Update data.json

### LINKED ISSUE #544: adityamangal
Profile

### PR DESCRIPTION
**Pull request by adityamangal1**
resolve #544

### CODE REVIEW

**Review by deephunt3r — APPROVED** [review]
LGTM 🚀

**Comment by skjha1** [comment]
LGTM

**Review by schmelto — APPROVED** [review]
LGTM could you resolve the merge conflict? If you need any help please let us know

**Review by vinzvinci — APPROVED** [review]
LGTM!

File: data.json [hunk 1]
```diff
@@ -970,5 +970,11 @@
     "githubUsername": "adityamangal1",
     "imageUrl": "https://user-images.githubusercontent.com/68494604/120438793-6bc1a000-c39f-11eb-845b-c145db5e2f9b.png",
     "issueNumber": 544
+ },
+ {
```
Comment (deephunt3r) [inline data.json:974]: @adityamangal1 May I know why you added @skjha1 details 🤔

**Review by deephunt3r — CHANGES_REQUESTED** [review]

File: data.json (hunk 1 above)
Comment (vinzvinci) [reply data.json:974]: Oops. I also notice. the imageUrl is wrong.

Comment (adityamangal1) [reply data.json:974]: I think the image URL is right can you recheck it?

Comment (deephunt3r) [reply data.json:974]: > I think the image URL is right can you recheck it? @vinzvinci

your info is correct but remove the @skjha1 information 
do you need any help?

Comment (adityamangal1) [reply data.json:968]: Yes I need some help @deephunt3r

File: data.json [hunk 2]
```diff
@@ -964,5 +964,11 @@
     "githubUsername": "gagan3012",
     "imageUrl": "https://user-images.githubusercontent.com/49101362/120364787-7c501880-c2c2-11eb-8f18-0f0ef10c4439.png",
     "issueNumber": 539
+  },
+  {
+    "name": "Aditya Mangal",
+    "githubUsername": "adityamangal1",
+    "imageUrl": "https://user-images.githubusercontent.com/68494604/120438793-6bc1a000-c39f-11eb-845b-c145db5e2f9b.png",
+    "issueNumber": 544
```
Comment (skjha1) [inline data.json:978]: LGTM

File: data.json (hunk 1 above)
Comment (deephunt3r) [reply data.json:968]: Yeah! What can I do for you

Comment (adityamangal1) [reply data.json:968]: Okay i will re do the process then

**Review by skjha1 — COMMENTED** [review]
Any problem @adityamangal

File: data.json (hunk 2 above)
Comment (adityamangal1) [reply data.json:978]: I don't know what's the problem then now I created again a pull request. @skjha1

**Pull request closed without merging**